mm2src/coins/qrc20.rs

```
@@ -416,6 +432,69 @@ impl Qrc20Coin {
         })
     }
 
+    async fn add_delegation(
+        &self,
+        amount: BigDecimal,
+        to_addr: Address,
+        fee: u64,
+    ) -> Result<GenerateQrc20TxResult, MmError<GenerateTxError>> {
+        let staker_address_hex = qtum::contract_addr_from_utxo_addr(to_addr.clone());
```

The amount cannot be < 100, otherwise, we need to return an error.